src/store/request-password-reset/saga.ts

```diff
@@ -43,6 +43,10 @@ export function* requestPasswordReset() {
 export function* watchRequestPasswordReset() {
   while (true) {
     yield take(SagaActionTypes.EnterRequestPasswordResetPage);
+
+    // set the stage back to the initial stage
+    yield put(setStage(RequestPasswordResetStage.SubmitEmail));
```

Comment:
Does `RequestPasswordResetStage.SubmitEmail` mean that the email is "submitted" OR "about to be submitted"? Wondering from the comment you had on line 47 🤔

Reply:
`RequestPasswordResetStage.SubmitEmail` means the Submit Email stage - 'about to be submitted'.

Two stages for the Request Password Reset Stage:

1. Submit Email - user is required to submit an email address.
2. Done - user has submitted the email and the request password reset is done / complete.

Reply:
Can we rename `SubmitEmail` to `RequestEmail`/`RequestSubmitEmail`? I think that would be more clear, the stage wasn't clear to me upon first look

Reply:
I think `RequestPasswordResetStage. RequestEmail` & `RequestPasswordResetStage. RequestSubmitEmail ` are also a little confusing. 

How about `RequestPasswordResetStage.EnterEmail` ?

Reply:
`RequestPasswordResetStage.EnterEmail` sounds good. But..how are `RequestEmail` & `SubmitEmail` different?